WordPress/Classes/System/WordPressAppDelegate+openURL.swift

```diff
@@ -0,0 +1,132 @@
+import WordPressAuthenticator
+
+/// Custom pattern matcher used for our URL scheme
+private func ~=(pattern: String, value: URL) -> Bool {
```

Comment:
Could we maybe move all these helpers / extensions to the bottom of the file? I think it can be easier to read the 'headlines' first and then move towards these details at the bottom after they're used.

Reply:
Works for me 😀